bin/node/runtime/Cargo.toml

```diff
@@ -83,7 +83,6 @@ pallet-treasury = { version = "3.0.0", default-features = false, path = "../../.
 pallet-utility = { version = "3.0.0", default-features = false, path = "../../../frame/utility" }
 pallet-transaction-payment = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-payment" }
 pallet-transaction-payment-rpc-runtime-api = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-payment/rpc/runtime-api/" }
-pallet-transaction-storage = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-storage" }
```

Comment:
A pallet for on-chain storage. [Test log link](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#cee512e48b8d4cea8fad90521ce0cf54)